internal/cli/cmd/install.go

```diff
@@ -266,6 +266,11 @@ cilium install
 cilium install --context kind-cluster1 --set cluster.id=1 --set cluster.name=cluster1
 `,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Flags().Visit(func(f *pflag.Flag) {
+				if f.Name == "set" && strings.Contains(f.Value.String(), "kubeProxyReplacement") {
```

Comment:
This isn't sufficient as a test for `kubeProxyReplacement`, as the value could be `kubeProxyReplacement=false`. In this case we do not want to set `params.UserSetKubeProxyReplacement = true`.